doc/final_report.md

```diff
@@ -0,0 +1,90 @@
+
+# Methods 
+
+## Data 
+
+The data set used for this study is the Offender Profile from 2017-2018 released by the Correctional Service of Canada. The link to this site can be found [here](https://open.canada.ca/data/en/dataset/844ff1e3-e137-41be-9ebe-6bd9843c1a53). Each entry in the dataset corresponds to a single offender serving a two or more year long sentence. The demographic details such as age, gender and marital status at year end are provided for each entry. This was retrieved from the Offender Management System (OMS).
+
+##  Analysis
+
+A hypothesis test with a significance level of 0.05 was carried out to determine if there was a statistically significant difference in the medians of the sentence length between the indigenous and non-indigenous groups. The features used from the data set to complete this analysis were the racial grouping, which was already separated into indigenous and non-indigenous, and aggregate sentence length. The median was selected as the measure of central tendency and the permutation test under the null hypothesis. The null hypothesis stated that there was no difference in the population medians in the sentence length between indigenous and non-indigenous inmates. The alternate hypothesis stated that there was a difference in the population median between indigenous and non-indigenous inmates.
+
+The R and Python programming languages [@R; @Python] and the following R and Python packages were used to perform the analysis:  docopt [@docopt],  knitr [@knitr], tidyverse [@tidyverse], infer[@infer], docopt [@docoptpython]. The code used to perform the analysis and create this report can be found here: https://github.com/UBC-MDS/DSCI_522_inference_on_indigenous_vs_non_indigenous_sentence_length_differences.
+
+
+# Results and discussion 
+
+Firstly, the only relevant features for our hypothesis test were the ‘aggregate sentence length’ and the ‘racial grouping column’. We decided to use the ‘racial grouping’ column over the race column as it already contained two groups as required by the hypothesis testing: ‘indigenous’ and ‘non-indigenous’. All other columns were dropped. The count of observations in either group was `r COUNT_IND` and `r COUNT_NON_IND`, meaning we had a large enough sample size to carry out a t-test if need be. 
+
+```{r observations_group, echo=FALSE}
+knitr::kable(model_quality$table, caption = "Table 1. Number of observations in each group.")
+```
+
+
+To decide what kind of measure of central tendency to apply in our hypothesis test, we looked at the distribution of the sentence lengths of either group (COLORS) using a density plot t. As the distribution for both groups was right skewed, we decided to use the median as our parameter as it is less affected by extreme values as It based on the count.  
```

Comment:
To decide what kind of measure of central tendency to apply in our hypothesis test, we looked at the distribution of the sentence lengths of either group (COLORS) using a density plot t. The distribution for both groups was right skewed, so we decided to use the median as our parameter as it is less affected by the outliers.